cnd/src/network/mod.rs

```diff
@@ -53,7 +55,9 @@ pub struct ComitNode<TSubstream> {
     #[behaviour(ignore)]
     pub db: Sqlite,
     #[behaviour(ignore)]
-    response_channels: HashMap<SwapId, oneshot::Sender<Response>>,
+    response_channels: Arc<Mutex<HashMap<SwapId, oneshot::Sender<Response>>>>,
+    #[behaviour(ignore)]
+    task_executor: TaskExecutor,
```

Comment:
Looks like it's used in https://github.com/comit-network/comit-rs/pull/1666/commits/0e58fe45799c06e0492ded780cf29710885c7778 (ie, before it's introduced).